pfioh/pfioh.py

```diff
@@ -263,7 +287,41 @@ def log_message(self, format, *args):
         return
 
     def do_GET(self):
+        isAuthorized, error = self.authorizeRequest()
+        if isAuthorized:
+            return self.execute_GET()
+        else:
+            d_ret = {
```

Comment:
This looks like the same code from do_POST

Reply:
No, it runs execute_GET instead of execute_POST (Emilio's code)

Reply:
I am talking about the else block.  It looks like there is a good use case for code reuse in a shared method.